packages/react-native-callkeep/src/withCallkeep.ts

```diff
@@ -130,7 +130,7 @@ export const withXcodeLinkBinaryWithLibraries: ConfigPlugin<{
 }> = (config, { library, status }) => {
   return withXcodeProject(config, (config) => {
     const options =
-      status === "optional" ? { settings: { ATTRIBUTES: ["Weak"] } } : {};
+      status === "optional" ? { weak: true } : {};
```

Comment:
So this changes no functionality then?

Reply:
Not at all! This has the same behaviour as before.